cfpb/consumerfinance.gov
# Add "Research Hub" pages to "Recent Updates" page

Currently the "Recent Updates" page at
https://www.consumerfinance.gov/activity-log/ does not include research reports from the "Research Hub" page at
https://www.consumerfinance.gov/data-research/research-hub/. Per internal D&CP#281, we want to add these pages there.

This change does so, adding a new "Office of Research Publication" option to the category dropdown on the Recent Updates page. These pages appear using the "chart" icon, the same way Report pages do.

## How to test this PR

Run a local server and visit http://localhost:8000/activity-log/, comparing it against https://www.consumerfinance.gov/activity-log/. You'll notice a new option in the category dropdown:

![image](https://github.com/cfpb/consumerfinance.gov/assets/654645/d078257f-47b1-4934-aa1e-42e6b5d22d4b)

If you search that category at http://localhost:8000/activity-log/?title=&categories=research-publication&from_date=&to_date=, the results will only include pages tagged with an ORP subcategory:

![image](https://github.com/cfpb/consumerfinance.gov/assets/654645/7cebc4c6-e375-4c68-ba34-50473998687f)

Note the inclusion of one "Report" is because that page is tagged with multiple categories.

One addition change is that ORP pages will now show the "chart" icon in their introduction; compare http://localhost:8000/data-research/research-reports/consumer-finances-in-rural-areas-of-the-southern-region/ to https://www.consumerfinance.gov/data-research/research-reports/consumer-finances-in-rural-areas-of-the-southern-region/.

Also visit the Research Hub page locally http://localhost:8000/data-research/research-hub/ and compare against production https://www.consumerfinance.gov/data-research/research-hub/; there are no changes.

However, if you click through the ORP topic tag on one of the results, it takes you to the filtered Reports page, again locally we will now show the icon http://localhost:8000/data-research/research-reports/?topics=office-of-research-publication vs. in production where we don't https://www.consumerfinance.gov/data-research/research-reports/?topics=office-of-research-publication.

## Checklist
